com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerModelLoader.cs

```diff
@@ -23,53 +26,92 @@ public static class ControllerModelLoader
         private static Dictionary<ulong, GameObject> controllerModelDictionary = new Dictionary<ulong, GameObject>();
 
         /// <summary>
-        /// Tries to load the controller model game object from the provided OpenXR ControllerModel.
+        /// A dictionary which caches the controller model gameobject associated with a specified input device
+        /// Stores a boolean indicating whether a warning was recently raised when trying to get the model key for this input device.
+        /// </summary>
+        private static Dictionary<InputDevice, bool> warningCache = new Dictionary<InputDevice, bool>();
+
+        /// <summary>
+        /// A dictionary which caches the controller model gameobject associated with a specified input device
+        /// Stores a boolean indicating whether an error was recently raised when trying to get the model for this model key.
```

Comment:
I think these summaries need updating (at least, the first lines of each)

Reply:
Removed the first line of each comment.